src/main/java/com/notably/logic/parser/DeleteCommandParser.java

```diff
@@ -16,6 +16,10 @@
  * Parses input arguments and creates a new DeleteCommand object
  */
 public class DeleteCommandParser implements CommandParser<DeleteCommand> {
+    private static final String ERROR_EMPTY_PATH = "A empty Path is detected please enter a valid Path. "
```

Comment:
Maybe we can just say "Path cannot be empty. To see.."?

Reply:
Should i redirect them to `help` ?  Not really sure what kind of information will be available there show the user a link of the UG, or does it condense some of the info ?

Reply:
I don't think you need to redirect to help tbh, just my thoughts!